cfg/LocalRef.h

```diff
@@ -0,0 +1,47 @@
+#ifndef SORBET_CFG_LOCALREF_H
+#define SORBET_CFG_LOCALREF_H
+
+#include "core/LocalVariable.h"
+
+namespace sorbet::cfg {
+class CFG;
+
+class LocalRef final {
+    int _id;
+
+public:
+    LocalRef() : _id(0){};
+    LocalRef(int id) : _id(id){};
```

Comment:
Could you make this constructor `explicit`? It would be good to avoid things like `LocalRef foo = 10`.

Reply:
TIL; is this compatible with `emplace` promoting an int to a `LocalRef`?

Reply:
I believe so, yes -- emplace_back just forwards arguments to the constructor.

Reply:
Turns out the answer is no! `emplace_back` failed with this marked as explicit.

Reply:
Well that sucks, boo emplace_back!